### ISSUE TITLE
In the Properties editor, initial color should match actual Vision color 

### ISSUE THREAD

**Issue by pklimai** [mipt-npm/visionforge#74]
Currently, in the "gravity" demo, the initial color is always set to black:

![grav](https://user-images.githubusercontent.com/10626378/150686322-b0690ccf-b9ff-468b-a402-625f4e820a20.png)

The "spheres" demo does not have this problem, by the way.

**Comment by altavir** [comment]
Yes, there is a problem with inferring the default color. I do not know how to fight it consistently. Thanks for the report.
